src/NeighborhoodCard/NeighborhoodCard.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './NeighborhoodCard.css';
+
+// NeighborhoodsCard - Name, picture, view listings btn 
+// Listings card - Name, picture, view details btn, favorite icon
+// card render: const checked = isFavorite, render checkbox as checked
+
+const NeighborhoodCard = (props) => {
+  console.log(props.listings)
```

Comment:
to avoid the props being written over and over maybe some de-structuring. 
const { id, areaNickname, name, location, description } = props

Reply:
Thanks for the ideas, Kyle. We went over these changes together and incorporated them. Please take a peek and merge if you think we're all good.